### PR TITLE
Provide permission to access old files to everyone

### PR DESCRIPTION
**Pull request by jialijelly**
For example in the case of old log files, it is not always the user/group will process it. Sometimes others also need it, thus granting access of old log files to be readable for everyone.

### CODE REVIEW

**Comment by astaxie** [comment]
--- FAIL: TestFileRotate_06 (0.00s)
	file_test.go:181: rotate file mode error

**Comment by jialijelly** [comment]
@astaxie  Thanks for the help! When can we see this changes in the next released build?

**Comment by jialijelly** [comment]
@astaxie  Its been a week. May I know when will this changes be merged and new build to be released?

**Comment by astaxie** [comment]
@jialijelly I think It would be better to add a config for this. Like `RotatePerm`. So it could be configurable.

**Comment by jialijelly** [comment]
I am not too sure, could you provide more information about the config??

**Comment by astaxie** [comment]
I think we can add another param here: https://github.com/astaxie/beego/blob/master/logs/file.go#L57

**Comment by jialijelly** [comment]
Okay sure! So i remove my current changes and add another param, okay?

**Comment by astaxie** [comment]
yea, works for me

**Comment by jialijelly** [comment]
@astaxie Hi, I have made the changes, please include this change in next build. Thank you!

File: logs/file.go [hunk 1]
```diff
@@ -56,17 +56,20 @@ type fileLogWriter struct {
 
 	Perm string `json:"perm"`
 
+	OtherPerm string `json:"otherperm"`
```
Comment (astaxie) [inline logs/file.go]: How about `RotatePerm`?

**Comment by jialijelly** [comment]
@astaxie  Hi, you mean to rename it to RotatePerm? If so, I've changed it

**Comment by astaxie** [comment]
@jialijelly yea

**Comment by jialijelly** [comment]
Hi, when will this changes be in the next released build?

**Comment by jialijelly** [comment]
Hi @astaxie , when will develop branch be merged to master branch?

**Comment by jialijelly** [comment]
Hi @astaxie , any updates when develop branch will merged to master?

**Pull request merged**